src/state/slices/portfolioSlice/utils.test.ts

```diff
@@ -1,13 +1,203 @@
+import {
+  assetIds,
+  btcAddresses,
+  btcCaip2,
+  btcCaip19,
+  ethCaip10s,
+  ethCaip19,
+  ethPubKeys,
+  foxCaip19,
+  mockBtcAccount,
+  mockBtcAddress,
+  mockEthAccount,
+  mockEthToken,
+  unknown1Caip19,
+  unknown2Caip19,
+  unknown3Caip19,
+  usdcCaip19,
+  yvusdcCaip19,
+  zeroCaip19
+} from 'test/mocks/accounts'
+import { bn } from 'lib/bignumber/bignumber'
+
+import { Portfolio } from './portfolioSlice'
 import {
   accountIdToChainId,
   accountIdToLabel,
   accountIdToSpecifier,
+  accountToPortfolio,
   assetIdtoChainId,
   btcChainId,
   ethChainId,
   findAccountsByAssetId
 } from './utils'
 
+const tokenBalance = (ethAccount: any, caip19: any) => {
+  return ethAccount.chainSpecific.tokens.find((token: any) => token.caip19 === caip19).balance
+}
+
+const ethAccount1 = mockEthAccount({
+  balance: '27803816548287370',
+  chainSpecific: {
+    nonce: 5,
+    tokens: [
+      mockEthToken({ balance: '42729243327349401946', caip19: foxCaip19 }),
+      mockEthToken({ balance: '41208456', caip19: usdcCaip19 }),
+      mockEthToken({ balance: '8178352', caip19: yvusdcCaip19 })
+    ]
+  },
+  pubkey: ethPubKeys[0]
+})
+
+const ethAccount2 = mockEthAccount({
+  balance: '23803816548287370',
+  chainSpecific: {
+    nonce: 5,
+    tokens: [
+      mockEthToken({ balance: '40729243327349401946', caip19: foxCaip19 }),
+      mockEthToken({ balance: '41208456', caip19: usdcCaip19 }),
+      mockEthToken({ balance: '8178352', caip19: yvusdcCaip19 })
+    ]
+  },
+  pubkey: ethPubKeys[1]
+})
+
+const ethAccount3 = mockEthAccount({
+  balance: '23803816548287371',
+  chainSpecific: {
+    nonce: 5,
+    tokens: [
+      mockEthToken({ balance: '4516123', caip19: unknown1Caip19 }),
+      mockEthToken({ balance: '8178312', caip19: yvusdcCaip19 }),
+      mockEthToken({ balance: '4516124', caip19: unknown2Caip19 }),
+      mockEthToken({ balance: '41208442', caip19: usdcCaip19 }),
+      mockEthToken({ balance: '4516125', caip19: unknown3Caip19 }),
+      mockEthToken({ balance: '40729243327349401958', caip19: foxCaip19 }),
+      mockEthToken({ balance: '4516126', caip19: zeroCaip19 })
+    ]
+  },
+  pubkey: ethPubKeys[2]
+})
+
+const btcAccount = mockBtcAccount({
+  balance: '1010',
+  chainSpecific: {
+    addresses: [
+      mockBtcAddress({ balance: '1000', pubkey: btcAddresses[0] }),
+      mockBtcAddress({ balance: '0', pubkey: btcAddresses[1] }),
+      mockBtcAddress({ balance: '10', pubkey: btcAddresses[2] }),
+      mockBtcAddress({ balance: '0', pubkey: btcAddresses[3] })
+    ]
+  }
+})
+
+const ethAccountSpecifier1 = ethCaip10s[0].toLowerCase()
+const ethAccountSpecifier2 = ethCaip10s[1].toLowerCase()
+const ethAccountSpecifier3 = ethCaip10s[2].toLowerCase()
+const btcAccountSpecifier = `${btcCaip2}:${btcAccount.pubkey}`
+
+const expectedPortfolio1: Portfolio = {
```

Comment:
After our test refactor, we're moving away from building portfolio object literals and using snapshots for the expected values. You can look in `portolioSlice.test.ts` for reference.

Reply:
Done ! Moved my tests from `utils.test.ts` to `portfolioSlice.test.ts`